src/main/java/org/osiam/resources/controller/UserController.java

```diff
@@ -25,35 +25,31 @@
 
 import org.osiam.auth.token.TokenService;
 import org.osiam.resources.helper.AttributesRemovalHelper;
-import org.osiam.resources.helper.JsonInputValidator;
-import org.osiam.resources.helper.RequestParamHelper;
 import org.osiam.resources.provisioning.SCIMUserProvisioning;
-import org.osiam.resources.scim.Meta;
 import org.osiam.resources.scim.SCIMSearchResult;
 import org.osiam.resources.scim.User;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.transaction.annotation.Transactional;
 import org.springframework.web.bind.annotation.*;
-import org.springframework.web.util.UriTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
 
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
+import javax.validation.Valid;
 import java.io.IOException;
 import java.net.URI;
 import java.util.Map;
 
 /**
  * This Controller is used to manage User
- * <p/>
+ * <p>
  * http://tools.ietf.org/html/draft-ietf-scim-core-schema-00#section-6
- * <p/>
+ * <p>
  * it is based on the SCIM 2.0 API Specification:
- * <p/>
+ * <p>
  * http://tools.ietf.org/html/draft-ietf-scim-api-00#section-3
- *
- *
  */
 @Controller
```

Comment:
Do want to make this a `@RestController`, too, and remove all the `@ResponseBody`s below?